examples/helm/with-open-webui/README.md

```
@@ -0,0 +1,99 @@
+# Open WebUI + computer-use-server on Kubernetes
+
+This is the Kubernetes analog of running `docker-compose.yml` + `docker-compose.webui.yml` together. You install two Helm charts side-by-side in the same namespace and wire them through a shared Secret.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)